modules/auth/proc-authenticationg-with-the-guest-user.adoc

```diff
@@ -0,0 +1,15 @@
+[id="authenticating-with-the-guest-user"]
```

Comment:
Is :_mod-docs-content-type: PROCEDURE not used for RHDH?

Reply:
```suggestion
:_mod-docs-content-type: PROCEDURE
[id="authenticating-with-the-guest-user_{context}"]
```